Failed to load library libortextensions.so

Hi, I wanted to try this awesome project and write an article about it, but I encountered an issue.

I compiled the onnxruntime-extensions on Linux and copied the library (actually everything, as it wasn't working) to the sd4j directory which contains your git repository:

```
sd4j$ ls -1
CONTRIBUTING.md
images
libnoexcep_operators.a
libocos_operators.a
libortcustomops.a
libortextensions.so
libortextensions.so.0
libortextensions.so.0.10.0
LICENSE.txt
pom.xml
README.md
SECURITY.md
src
target
text_tokenizer
THIRD_PARTY_LICENSES.txt
```

Then I run:

`sd4j$ mvn package exec:exec -DmodelPath=../stable-diffusion-v1-5/`

And I get the following error:

```
[INFO] --- exec-maven-plugin:3.1.0:exec (default-cli) @ sd4j ---
Gtk-Message: 21:05:49.827: Failed to load module "canberra-gtk-module"
Exception in thread "main" java.lang.IllegalStateException: Failed to instantiate SD4J pipeline
	at com.oracle.labs.mlrg.sd4j.SD4J.factory(SD4J.java:356)
	at com.oracle.labs.mlrg.sd4j.SD4JApp.<init>(SD4JApp.java:89)
	at com.oracle.labs.mlrg.sd4j.SD4JApp.main(SD4JApp.java:421)
Caused by: ai.onnxruntime.OrtException: Error code - ORT_FAIL - message: Failed to load library libortextensions.so with error: libortextensions.so: kan gedeeld objectbestand niet openen: Bestand of map bestaat niet
	at ai.onnxruntime.OrtSession$SessionOptions.registerCustomOpLibrary(Native Method)
	at ai.onnxruntime.OrtSession$SessionOptions.registerCustomOpLibrary(OrtSession.java:700)
	at com.oracle.labs.mlrg.sd4j.TextEmbedder.<init>(TextEmbedder.java:113)
	at com.oracle.labs.mlrg.sd4j.SD4J.factory(SD4J.java:340)
	... 2 more
[ERROR] Command execution failed.
```

I looked at the code and the TextEmbedder seems to not even load the libortextensions but ortextensions:

`        this.tokenizerOpts.registerCustomOpLibrary(System.mapLibraryName("ortextensions"));`

Probably I'm doing something stupid here, but I couldn't quickly figure out what, so any help/tips would be really appreciated.

--- a/src/main/java/com/oracle/labs/mlrg/sd4j/TextEmbedder.java
+++ b/src/main/java/com/oracle/labs/mlrg/sd4j/TextEmbedder.java
@@ -110,7 +110,7 @@ public final class TextEmbedder implements AutoCloseable {
     public TextEmbedder(Path tokenizerPath, Path embedderPath, OrtSession.SessionOptions embedderOpts) throws OrtException {
         this.env = OrtEnvironment.getEnvironment();
         this.tokenizerOpts = new OrtSession.SessionOptions();
-        this.tokenizerOpts.registerCustomOpLibrary(System.mapLibraryName("ortextensions"));
+        this.tokenizerOpts.registerCustomOpLibrary("./"+System.mapLibraryName("ortextensions"));
         this.tokenizer = env.createSession(tokenizerPath.toString(), tokenizerOpts);
         this.textEmbedderOpts = embedderOpts;
         this.textEmbedder = env.createSession(embedderPath.toString(), textEmbedderOpts);

